lib/ansible/module_utils/k8s/inventory.py

```diff
@@ -212,7 +212,8 @@ def get_services_for_namespace(self, name, namespace):
             ports = [{'name': port.name,
                       'port': port.port,
                       'protocol': port.protocol,
-                      'targetPort': port.target_port} for port in service.spec.ports]
+                      'targetPort': port.target_port,
+                      'nodePort': port.node_port} for port in service.spec.ports]
```

Comment:
I think, it would look better readable if you moved `for ...` part to separate line.